dask_expr/_collection.py

```diff
@@ -1363,6 +1363,72 @@ def iloc(self):
 
         return ILocIndexer(self)
 
+    def categorize(self, columns=None, index=None, split_every=None, **kwargs):
+        """Convert columns of the DataFrame to category dtype.
```

Comment:
I've noticed we seldom add docstrings. What's the current approach here?

Reply:
We haven't generated docs yet, so I didn't bother with docstrings. We have to figure docs out soonish though, so feel free to add them